rcl/test/rcl/test_get_type_description_service.cpp

```diff
@@ -85,7 +85,56 @@ static bool service_exists(
       }
     }
 
-    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    std::this_thread::sleep_for(std::chrono::milliseconds(10));
```

Comment:
(nice to have) adding argument `std::chrono::milliseconds period` would be more flexible.

Reply:
> (nice to have) adding argument `std::chrono::milliseconds period` would be more flexible.

Yeah, that's true.  Given that this is only used in this test, it would be easy enough to add later if we need the flexibility.  So I'm going to not do that for now, but thanks for the idea.